components/gantry/multiaxis/multiaxis.go

```diff
@@ -133,7 +151,8 @@ func (g *multiAxis) GoToInputs(ctx context.Context, goal []referenceframe.Input)
 	ctx, done := g.opMgr.New(ctx)
 	defer done()
 
-	return g.MoveToPosition(ctx, referenceframe.InputsToFloats(goal), nil)
+	speeds := []float64{}
```

Comment:
Keep a note here about passing in the empty float, we'll have to discuss what this should look like if this eventually gets called by a motion planner.